aria-practices.html

```diff
@@ -1902,14 +1902,46 @@ <h4>Keyboard Interaction</h4>
         <h4>WAI-ARIA Roles, States, and Properties</h4>
         <ul>
           <li>An element that contains or owns all the listbox options has role <a href="#listbox" class="role-reference">listbox</a>.</li>
-          <li>Each option in the listbox has role <a href="#option" class="role-reference">option</a> and is a DOM  descendant of the element with role <code>listbox</code> or is referenced by an <a href="#aria-owns" class="property-reference">aria-owns</a> property on the listbox element.</li>
-          <li>If the listbox is not part of another widget, then it has a visible label referenced by <a href="#aria-labelledby" class="property-reference">aria-labelledby</a> on the element with role <code>listbox</code>.</li>
-          <li>In a single-select listbox, the selected option has <a href="#aria-selected" class="property-reference">aria-selected</a> set to <code>true</code>. </li>
-          <li>if the listbox supports multiple selection:
+          <li>
+            Each option in the listbox has role <a href="#option" class="role-reference">option</a> and is contained in or owned by either:
+            <ul>
+              <li>The element with role <code>listbox</code>.</li>
+              <li>An element with role <a href="#group" class="role-reference">group</a> that is contained in or owned by the element with role <code>listbox</code>.</li>
+            </ul>
+          </li>
+          <li>
+            Options contained in a <code>group</code> are referred to as <q>grouped options</q> and form what is called an <q>option group.</q>
+            If a listbox contains grouped options, then:
             <ul>
-              <li> The element with role <code>listbox</code> has <a href="#aria-multiselectable" class="property-reference">aria-multiselectable</a> set to <code>true</code>. </li>
-              <li>All selected options have <a href="#aria-selected" class="state-reference">aria-selected</a> set to <code>true</code>. </li>
-              <li>All options that are not selected have <a href="#aria-selected" class="state-reference">aria-selected</a> set to <code>false</code>. </li>
+              <li>The listbox contains at least two option groups.</li>
```

Comment:
I agree with @jnurthen for the potential case of lisbox contains only one option group.

Reply:
This would actually end up happening a lot when used in a combo box and the entries get reduced based on input in the combo box. This constraint should be dropped as should the next one.